src/sagemaker/local_run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
can we move this in our folder? `-/serve/`??

Reply:
Yup that is possible, ideally we can remove this file

Reply:
Okay, let's remove this one as I saw [src/sagemaker/serve/local_run.sh](https://github.com/aws/sagemaker-python-sdk/pull/4808/files#diff-65269e0167430bd667d4a697e51f5f35b08af5ba1c56ab68d7fe45458cddb2a0)